File: 240507/범위 안의 수 2/number-in-range-2.js
const fs = require('fs');
const nums = fs.readFileSync(0).toString().trim().split('\n').map(Number);

let cnt = 0;
let sum = 0;

nums.forEach((num)=>{
    if(num >= 0 && num <= 200){
        cnt++;
        sum += num;
    }
})

console.log(sum, (sum/cnt).toFixed(1));

File: 240507/범위 안의 수 2/number-in-range-2.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cnt = 0;
        int sum = 0;

        for(int i = 0; i < 10; i++){
            int num = sc.nextInt();

            if(num >= 0 && num <= 200){
                cnt++;
                sum += num;
            }
        }

        System.out.printf("%d %.1f", sum, (double)sum / cnt);
    }
}